.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+env:
+  ANKI_TEST_REVISION: ed8340a4e3a2006d6285d7adf9b136c735ba2085
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu, macos, windows]
+        py: [3.6, 3.8, 3.9]
+        include:
+          - os: ubuntu
+            py-cmd: python
+          - os: macos
+            py-cmd: python3
+          - os: windows
+            py-cmd: python
+        exclude:
+          # compatible pyaudio not available
+          - os: windows
+            py: 3.8
+          - os: windows
+            py: 3.9
+    steps:
+      - uses: actions/checkout@v2
+      - if: ${{ matrix.os == 'ubuntu' }}
+        run: sudo apt-get install portaudio19-dev python3-dev
+      - if: ${{ matrix.os == 'macos' }}
+        run: brew install portaudio
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.py }}
+      - run: ${{ matrix.py-cmd }} -m pip install --upgrade pip wheel setuptools
+      - if: ${{ matrix.os == 'windows' }}
+        run: ${{ matrix.py-cmd }} -m pip install pywin32
+      - run: ${{ matrix.py-cmd }} -m pip install -e ".[test]" pytest-cov codecov
```

Comment:
Could you add a comment like

    # The ".[test]" syntax installs the packages in install_requires and tests_require from setup.py. See https://stackoverflow.com/a/43893575

Reply:
added the comment, but the full URL, etc. seems a bit overkill... if anything, i'd recommend normalizing the `test_require`, which is deprecated, to use the `extras_require` syntax.